docs/functionality/event-data.md

```diff
@@ -0,0 +1,221 @@
+## Supported Event Data Providers
+
+- [DataFactory](#datafactory)
+- [Metrica](#metrica)
+- [Opta](#opta)
+- [Sportec](#sportec)
+- [SportsCode](#sportscode)
+- [StatsBomb](#statsbomb)
+- [WyScout](#wyscout)
+
+
+### DataFactory
+
+#### load
+`kloppy.kloppy._providers.datafactory.load(event_data, event_types=None, coordinates=None, event_factory=None)`
```

Comment:
Is it possible to generate `event-data.md` and `tracking-data.md` from code? And if not, please make sure the function call looks like `kloppy.datafactory.load`. We try to hide the `_providers` module from the end-users.

Reply:
It's not generated from code. And yes, good point. I'll update that.